CHANGELOG.md

```diff
@@ -15,23 +15,27 @@ We use the following categories for changes:
 ## [Unreleased]
 
 ### Added
-- Add ability to configure chunk interval on startup
+- Add ability to configure chunk interval on startup [#991]
+- Add `ps_trace.delete_all_traces()` function to delete all trace data [#1012]
+- Add `ps_trace.set_span_retention_period(INTERVAL)` function to set span retention period. [#1015]
+- Add `ps_trace.get_span_retention_period()` database function to get current span retention period. [#1015]
+- BREAKING: Enable and configure 30 day default retention period for span data. [#1015]
 
 ### Changed
-- Rename CLI flags to improve user interface
+- Rename CLI flags to improve user interface [#964]
```

Comment:
This is technically a breaking change, no?

Reply:
Depends on how you define breaking change. The old CLI flags still work, so there's no breakage when you update to promscale 0.8.0.

Reply:
Ah, it read to me like the old names no longer exist and they were given new names - if both sets still exist, then yeah, no worries.

Reply:
👍